openapi-module/README.md

```diff
@@ -0,0 +1,159 @@
+# OpenAPI Module for Flexbench
+
+This module provides functionalities to parse OpenAPI documents and generate test scenarios for Flexbench.
+
+## Prerequisites
+
+- To use this module, you need to install Node.js. It is recommended to use the latest LTS version available.
+  Please install Node.js using pre-built installers for your platform to avoid incompatibility issues with different development tools.
+
+### To check that Node.js was installed correctly, type the following commands in your terminal client:
+
+```sh
+node -v
+npm -v
+```
+
+## Project Structure
+
+```
+openapi-module/
+├── node_modules/
+├── sample/
+│   └── sample-openapi.yaml
+├── scripts/
+│   ├── generate-all.js
+│   ├── generate-curl.js
+│   └── generate-flex.js
+├── src/
+│   ├── generators/
+│   │   ├── curl-generator.js
+│   │   ├── field-mapping.js
+│   │   ├── flex-generator.js
+│   │   └── gpt-flex-generator.js
+│   ├── GPT/
+│   │   └── config.js
+│   └── parsers/
+│       └── openapi-parser.js
+├── temp/(generated files will be saved under this folder)
+├── test/
+│   ├── generators.test.js
+│   └── parser.test.js
+├── .gitignore
+├── package-lock.json
+├── package.json
+└── README.md
+```
+
+## Installation
+
+### To install dependencies, run the following command:
+
+```sh
+npm install
+```
+
+## Usage
+
+### Before generating .flex files & cURL commands for Flexbench by parsing your OpenAPI YAML file:
+
+You need to decide whether to generate scenarios using AI or a static approach:
+
+#### Using the Static Approach (Faker)
+
+1. This approach uses configurable templates with Faker to generate fake data for each schema and field.
+2. To use this method, go to `src/GPT/config.js` and set `useGPT` to `false`.
+
+#### Using the AI Approach (OpenAI API)
+
+1. This approach uses configurable templates with GPT prompts to generate fake data for the .flex scenarios.
+2. To use this method, go to `src/GPT/config.js` and set `useGPT` to `true`.
```

Comment:
Haven't you changed the approach to use an argument --useGPT=true|false? Update README here with the correct info